src/content/pages/wasm.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: WebAssembly Summit
+subtitle: A summit to bring together maintainers and users of the Python with WebAssembly, to discuss the state of this ecosystem, existing challenges and ongoing work.
+---
+
+# WebAssembly Summit
+
+* **When**: Tuesday, July 9th.
+* **Where**: Prague Congress Centre (PCC), Room TBC
+* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Conference, Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
```

Comment:
```suggestion
* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
```
I don't think Conference is allowed as this is Wednesday to Friday only. @thepetk , @artcz could we please get an official answer?

Reply:
Well, I just copied the advice I was given for last year. Remember, this is a technical summit **for the WASM community**, not a tutorial. My vote is to cast the net as wide as possible.

Reply:
Any ticket is fine. Has been confirmed.